src/DynamoCore/Configuration/DebugModes.cs

```diff
@@ -35,32 +34,30 @@ public class DebugMode
             public bool IsEnabled;
         }
 
-        private static void AddDebugMode(string name, string description)
+        private static void AddDebugMode(string name, string description, bool isEnabled = false)
         {
             debugModes[name] = new DebugMode()
             {
                 Name = name,
                 Description = description,
-                IsEnabled = false
+                IsEnabled = isEnabled
             };
         }
         private static void RegisterDebugModes()
         {
             // Register app wide new debug modes here.
-            AddDebugMode("Python3DebugMode", "Enable/disable Python3 Engine.");
+            AddDebugMode("Python3DebugMode", "Enable/disable Python3 Engine.", true);
+            AddDebugMode("Python2ObsoleteMode", "Enable/disable warnings regarding Python 2 obsoletion.");
         }
 
         internal static void LoadDebugModesStatusFromConfig(string configPath)
```

Comment:
is this method only used for testing?

Reply:
It is also used in normal initialization, for a file named `debug.config`. That's how Dynamo picks it up normally.